src/desktop/components/split_test/README.md

```diff
@@ -148,12 +148,13 @@ app.get("/:slug", async (req, res, next) => {
       const {
         ...
       } = await buildServerApp({
+        req,
+        res,
         routes,
-        url: req.url,
-        userAgent: req.header("User-Agent"),
         context: {
-          ...buildServerAppContext(req, res),
-          showMyTest: MY_TEST === "experiment"
+          injectedData: {
```

Comment:
One thing about adding this as a sub prop like this and then placing stuff inside is that `context` is already implicitly defined as a container for "injected" data (or dependency injection). So i think it might be adding a tiny level of unnecessary complexity, versus just doing `context: { foo: 'bar' }`

Reply:
The only existing uses of this added context in the codebase were already labelled as `injectedData`, so I typed it that way to at least keep things consistent. I wanted to keep this typed because we were inconsistently passing `mediator` and `user` when they were already present.